cumulus_etl/deid/mstool.py

```diff
@@ -80,6 +80,17 @@ def _compare_file_sizes(target: dict[str, int], current: dict[str, int]) -> floa
     return total_current / total_expected
 
 
+def _get_file_size_safe(path: str) -> int:
+    try:
+        return os.path.getsize(path)
+    except FileNotFoundError:
```

Comment:
(comment being written from an outsider's perspective, so forgive stupid questions plz)
Is this the right Error? The 3.12 docs I was looking at for `os.path` [suggest it would throw OSError](https://docs.python.org/3/library/os.path.html#os.path.getsize)

Reply:
Ah yeah as we mentioned on slack, it's a builtin subclass of OSError.